pkg/storage/monitoring.go

```diff
@@ -0,0 +1,22 @@
+package storage
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var (
+	pointsStored = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Namespace: "metrics_server",
+			Subsystem: "storage",
+			Name:      "point_total",
```

Comment:
According to Prometheus' best practices, the `_total` suffix is reserved to counter metrics. I would suggest changing to `metrics_server_storage_points`.